modules/oe_content_entity/modules/oe_content_entity_contact/config/post_updates/00007_node_reference_field/field.storage.oe_contact.oe_node_reference.yml

```diff
@@ -2,15 +2,15 @@ langcode: en
 status: true
```

Comment:
`  \Drupal::service('module_installer')->install(['entity_reference_revisions']);` can be removed from `oe_content_entity_contact_post_update_00007()`

Reply:
Good catch.